### PR TITLE
Translations services testing

### PR DESCRIPTION
**Pull request by danachen**
Includes:
[x] add translations services tests
[x] broke up a couple of services into smaller units
[x] re-wrote tests for broken services
[x] typing translations services
[x] add delete route for words
[x] add delete test for words route

### CODE REVIEW

File: src/tests/wordsRoutes.test.ts [hunk 1]
```diff
@@ -74,6 +73,14 @@ describe('Testing updating a word', () => {
   });
 });
 
+describe('Testing deleting a word', () => {
+  test('delete an existing word', async () => {
+    await api
+      .delete('/api/words/word/1')
+      .expect(200);
```
Comment (mrchrmn) [inline src/tests/wordsRoutes.test.ts:80]: Should expect `204`after deletion.

File: src/routes/words.ts [hunk 2]
```diff
@@ -49,4 +49,10 @@ router.put('/word/:wordId/user/:userId', async(req, res): Promise<void> => {
   res.send(updatedStatus);
 });
 
+router.delete('/word/:wordId', async(req, res): Promise<void> => {
+  const id = req.params.wordId;
+  const result = await words.remove(Number(id));
+  res.send(result);
```
Comment (mrchrmn) [inline src/routes/words.ts:55]: Should send `204` for deletion.

**Comment by mrchrmn** [comment]
The translation services SQL queries are frightening :) 

I am going to merge this but there will be conflicts later with my current pull request whereI also touched the words routes. 

Also, if you don't mind, once merged I would go in and reformat the SQL queries to multiple lines for better readability.

File: src/routes/words.ts (hunk 2 above)
Comment (danachen) [reply src/routes/words.ts:55]: ✅

**Pull request merged**